models/rawPg/nearbySchema.sql

```diff
@@ -14,8 +14,40 @@ CREATE TABLE Nearby (
   created_at timestamp without time zone default (now() at time zone 'utc')
 );
 
+CREATE TABLE IF NOT EXISTS Attraction (
+  id SERIAL PRIMARY KEY,
```

Comment:
you can have id here, but index another id somewhere else and use that for queries

Reply:
Yeah I will update and send another PR